app/models/whitehall_imported_asset.rb

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+
+# Represents the raw import of an asset from Whitehall Publisher and
+# the import status of the asset into Content Publisher
+class WhitehallImportedAsset < ApplicationRecord
```

Comment:
Might be an idea to namespace this class under whitehall_import (as we've done with file_attachment and image) which may reduce a proliferation of classes in models.

Saying this I am expecting we're going to have a WhitehallMigration model and I think it'd make sense to have both import and asset namespaced under that.